test/bdd/features/edv_interop_transmute.feature

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@edv_interop_transmute
+Feature: Demonstrating TrustBloc's EDV REST API interoperability with Transmute's implementation.
+
+  @data_vault_creation
+  Scenario: Use EDV client to check for interoperability.
```

Comment:
I would think we would show in the same feature file connecting to both our system and transmute?

Reply:
see https://github.com/trustbloc/edge-service/blob/master/test/bdd/features/thirdparty_verifier_e2e.feature#L18